mod_reforged/hooks/config/strings.nut

```diff
@@ -2058,7 +2067,7 @@ foreach (vanillaDesc in vanillaDescriptions)
 	}),
 	RF_SwordmasterJuggernaut = ::UPD.getDescription({
 		Fluff = "There\'s a fine line between bravery and stupidity.",
-		Requirement = "Two-Handed Sword"
+		Requirement = "Two-Handed Non-Fencing Sword"
```

Comment:
This change is reasonable but the perk's code doesn't coincide with this. So this should be extracted into a separate PR where we change the perk's behavior.